libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/api/RoadShieldsCallback.kt

```diff
@@ -1,36 +0,0 @@
-package com.mapbox.navigation.ui.maneuver.api
-
-import com.mapbox.navigation.ui.maneuver.model.Maneuver
-import com.mapbox.navigation.ui.maneuver.model.PrimaryManeuver
-import com.mapbox.navigation.ui.maneuver.model.RoadShield
-import com.mapbox.navigation.ui.maneuver.model.RoadShieldError
-import com.mapbox.navigation.ui.maneuver.model.SecondaryManeuver
-import com.mapbox.navigation.ui.maneuver.model.SubManeuver
-
-/**
- * An interface that is triggered when road shields are available.
- */
-fun interface RoadShieldsCallback {
```

Comment:
Based on above, this interface should stay.

Reply:
I don't think we need this interface as the new maneuver API should directly be able to use `RouteShieldCallback` provided by libnavui-shield module.